Sample/Sample/ViewController.swift

```diff
@@ -21,6 +21,7 @@ class ViewController: UIViewController {
 
         let videoURL = URL(fileURLWithPath: Bundle.main.path(forResource: "google-help-vr", ofType: "mp4")!)
         let player = AVPlayer(url: videoURL)
+        player.automaticallyWaitsToMinimizeStalling = false
```

Comment:
I guess we don't need to set on sample app you have already set `player.automaticallyWaitsToMinimizeStalling` inside library.